### PR TITLE
[FEAT] Customize the 404 page

### LINKED ISSUE #6: [FEAT] Add a custom 404 page
Once we have the first page (#2), create the 404 page

Image proposal

404_v02

Inspired from the bonitasoft website https://www.bonitasoft.com/404

404_bonita_page

### PR DESCRIPTION
**Pull request by csouchet**
Closes #6

### CODE REVIEW

File: src/pages/404.js [hunk 1]
```diff
@@ -1,51 +1,22 @@
...
-            <br />
-            Try creating a page in <code style={codeStyles}>src/pages/</code>.
-            <br />
-          </>
-        ) : null}
-        <br />
-        <Link to="/">Go home</Link>.
+      <h1>Page not found</h1>
+      <p>
+        Oops, looks like the page cannot be found...
```
Comment (aibcmars) [inline src/pages/404.js:11]: not sure we need this,
404 nowadays is very expressive I think :)

the header takes space and the 'go back' is not visible without scrolling
image

Comment (csouchet) [reply src/pages/404.js]: Now, it's sommething like 
image

**Review by tbouffard — COMMENTED** [review]
please remove all changes in the public directory.
This directory should not be committed as it contains generated resources

**Review by tbouffard — APPROVED** [review]
The scrolling bar is still displayed on my pc.
But this is fine as a first implementation as the image is fully displayed (nothing on the bottom on the image).

**Comment by csouchet** [comment]
> please remove all changes in the public directory.
> This directory should not be committed as it contains generated resources

I didn't see the favicon in other folders.

**Pull request merged**